test/dynamo/test_export.py

```diff
@@ -2037,7 +2037,9 @@ def forward(self, x):
         dynamo_result = out_graph(inp)
         self.assertEqual(dynamo_result, m(inp))
 
-    @config.patch(dynamic_shapes=True)
+    @config.patch(
+        dynamic_shapes=True,
+    )
```

Comment:
no op reformatting 🤔

Reply:
Linter did this, but I can try to squash it back